backend/domain/Room.js

```diff
@@ -64,6 +79,28 @@ class Room {
         this.roundNumber++;
     }
 
+    getWinningRole() {
+        const numOfMafiaAlive = this.getPlayersByRole(roles.MAFIA).filter((player) => player.isAlive);
+        const numOfJesterAlive = this.getPlayersByRole(roles.JESTER).filter((player) => player.isAlive);
+        const numOfCiviliansAlive = [
+            this.getPlayersByRole(roles.CIVILIAN),
+            this.getPlayersByRole(roles.MEDIC),
+            this.getPlayersByRole(roles.DETECTIVE),
+        ].filter((player) => player.isAlive);
```

Comment:
filter returns an array 
"The filter() method creates a new array with all elements that pass the test implemented by the provided function."
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/filter

Reply:
need .length at end
 for all of the numofXalive

Reply:
fixed